powerquery-docs/connectors/palantir-foundry-datasets.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Palantir Foundry Power Query
```

Comment:
nit: Palantir Foundry Power Query Connector

Reply:
The "connector" part of the title should be lowercase - Fixed in review